src/OpenTelemetry.Api/Trace/ActivityExtensions.cs

```diff
@@ -57,20 +57,26 @@ public static Status GetStatus(this Activity activity)
     }
 
     /// <summary>
-    /// Adds an activity event containing information from the specified exception.
+    /// Adds an <see cref="ActivityEvent"/>  containing information from the specified exception.
     /// </summary>
     /// <param name="activity">Activity instance.</param>
     /// <param name="ex">Exception to be recorded.</param>
+    /// <remarks> The exception is recorded as per <a href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/exceptions.md"/>.
```

Comment:
This sentence needs to be corrected. This would be rendered as:

The exception is recorded as per [. "exception.stacktrace" is represented using the value of ](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/exceptions.md)[Exception.ToString](https://learn.microsoft.com/dotnet/api/system.exception.tostring).

Reply:
You could change this to:

<remarks> The exception is recorded as per the <a href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/exceptions.md"/>specification.</a>
    /// "exception.stacktrace" is represented using the value of <a href="https://learn.microsoft.com/dotnet/api/system.exception.tostring">Exception.ToString</a> method.
   /// </remarks>